pyproject.toml

```diff
@@ -20,8 +20,9 @@ dependencies = [
 ]
 
 [tool.scikit-build]
-cmake.source-dir = "./"
 cmake.build-type = "Release"
+cmake.source-dir = "./"
+cmake.targets = ["py_generate_pipeline"]  # Adding genai would trigger a Release build and Debug build after it. py_generate_pipeline depends on genai and genai will be built anyway. It's not been investigated why both build types are triggered.
```

Comment:
maybe because of ninja multi-config which builds several build types at the same time? 
You can probably use the same https://github.com/openvinotoolkit/openvino_tokenizers/pull/162